docs/modules/provisioning-app/pages/cd-user.adoc

```diff
@@ -0,0 +1,17 @@
+:toc: macro
+
+= Syncing your Technical User With ODS Components
+
+When you create a new project, the provisioning app triggers a Jenkins pipeline. This pipeline adds `CD_USER_ID` as the name and `CD_USER_B64` as the encoded password to the secrets of your newly generated Openshift cd project. This is the technical user for your project. Every time a (cd) project is generated, these parameters are added to the secrets of your Openshift project. At this point however, you're still not ready to push some code into your repository or succesfully generate a new quickstarter using ods-shared-library. In order to achieve the two, you would manually need to add this same technical user to the repository settings of the two ODS repositories: `ods-jenkins-shared-library` and `ods-quickstarters`. Hence you have a technical user that will have read access to your project. 
```

Comment:
Typo `CD_USER_B64`

Reply:
> In order to achieve the two, you would manually need to add this same technical user to the repository settings of the two ODS repositories: `ods-jenkins-shared-library` and `ods-quickstarters`.

The prov app gives read access to the project specific cd user. See `READABLE_OPENDEVSTACK_REPOS`. Are you talking about the global cd user? I would discourage to use that.

Reply:
You're right. I meant to say `Hence you have a technical user that ODS components can refer to in order to access to your project`, as the project read permission alone isn't enough. Also could you point out where and how `READABLE_OPENDEVSTACK_REPOS` is used in prov-app?